src/test/java/com/wikia/webdriver/testcases/mercurytests/SignupTests.java

```diff
@@ -75,4 +75,17 @@ public void MercurySignupTest_005_signupErrorTooYoungUser() {
           .typeBirthdate("12", "12", "2009").register();
     mobileSignup.verifyBirthdateError();
   }
+
+  @Test(groups = {"MercurySignupTest_006"})
+  @Execute(onWikia = "ja.ja-test")
+  public void MercurySignupTest_006_japaneseUserSignup() {
```

Comment:
Yep, the small differences would come as method arguments, however don't agree about clearness.